src/components/style/forms/inputGroup.css

```diff
@@ -89,3 +72,20 @@
 .fdsInputGroup .fdsIconInput .fds-icon {
   z-index: 3;
 }
+
+/* Safari 10+ Only */
+@media not all and (min-resolution: 0.001dpcm) {
+  @media {
+    .fdsInputGroup select {
+      border-color: rgba(var(--rgb-charcoal), 0.24);
+      min-height: initial;
+      height: calc(var(--space-double) + 2px);
+      margin: -1px;
+      transform: translateX(1px);
+    }
+
+    .fdsInputGroup select:focus {
+      outline: 2px solid rgba(var(--rgb-aqua), 0.6);
+    }
+  }
+}
```

Comment:
Would we want to add in the comments above what this snippet is doing? I'm sort of confused what this is for.